packages/opentelemetry-resources/test/regression/existing-detectors-1-9-1.test.ts

```diff
@@ -20,7 +20,7 @@ import * as assert from 'assert';
 // If compilation fails at this point then the changes made are breaking.
 class RegressionTestResourceDetector_1_9_1 implements Detector {
   async detect(_config?: ResourceDetectionConfig): Promise<Resource> {
-    return Resource.empty();
+    return Resource.empty() as any;
```

Comment:
The comment above this detector still holds - if this needs to be modified, the changes in this PR are breaking.